src/lib/index.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable import/prefer-default-export */
 import AgGrid from './components/AgGrid.react';
+import {getApi, getColumnApi} from './fragments/AgGrid.react';
```

Comment:
Hmm this is awkward... we can't just import from the fragment or it ceases to be an async chunk.

Simplest thing that comes to mind: put the outer functions in `components/AgGrid.react` along with a container you can inject inner functions into:
```js
export const apiGetters = {};

const _get = (flavor) => (id) => {
    // optional chaining so before the fragment exists it'll just return undefined
    // which does the right thing because clearly no grid is initialized yet!
    const api = apiGetters[flavor]?.(id);
    if (api) {
        return api;
    }
    throw new Error(
        `no grid found, or grid is not initialized yet, with id: ${id}`
    );
};
export const getApi = _get('getApi');
export const getColumnApi = _get('getColumnApi');
```
Then in the fragment, inject the real API getters:
```js
import {apiGetters} from '../components/AgGrid.react'; // combine with propTypes and defaultProps import we already have

apiGetters.getApi = (id) => agGridRefs[stringifyId(id)]?.api;
apiGetters.getColumnApi = (id) => agGridRefs[stringifyId(id)]?.columnApi;
```